client/boot/index.js

```diff
@@ -88,6 +88,12 @@ function setUpContext( layout, reduxStore ) {
 
 		context.layout = layout;
 		context.store = reduxStore;
+		context.render = ( reactElement, domContainerNode ) => {
+			return ReactDom.render(
```

Comment:
If we intend to replace ReactDom.render, then we need to kinda return the same I think..